packages/portfolio-contract/test/portfolio.contract.test.ts

```diff
@@ -45,26 +111,56 @@ test('start my orch contract', async t => {
       }),
     ),
   );
+  return { common, zoe, started };
+};
 
-  const hookAddress = await E(myKit.publicFacet).getHookAddress();
-  t.log('hookAddress', hookAddress);
-  t.notThrows(() => mustMatch(hookAddress, ChainAddressShape));
+test('open portfolio with USDN position', async t => {
```

Comment:
_blocking_

We should also test that:
1. IBC transfer happens before swap + lock
2. swap + lock messages are sent thru the bridge